tests/tests.rs

```diff
@@ -1265,7 +1265,7 @@ fn from_same_bit_widths() {
 #[test]
 fn calculation_with_number_trait() {
     fn increment_by_1<T: num_traits::WrappingAdd + Number>(foo: T) -> T {
-        foo.wrapping_add(&T::new(1.into()))
+        foo.wrapping_add(&T::from_(1u8))
```

Comment:
In our tests, this is basically the API breakage compare to before. Not sure how common this pattern is for users

Reply:
Another way to solve is using `T::new(1.try_into(u8).unwrap())`, but that's kinda ugly as I couldn't figure out a way to ensure the error is actually Debug. So a regular unwrap() won't do it.